app/scripts/vendor/env-test.js

```diff
@@ -31,13 +31,8 @@
   var pwElement = document.createElement('input');
   pwElement.setAttribute('type', 'password');
 
-  try {
-    // IE8 will blow up here.
-    pwElement.setAttribute('type', 'text');
-    docElement.className += ' toggle-pw';
-  } catch(e) {
-    docElement.className += ' no-toggle-pw';
-  }
+  pwElement.setAttribute('type', 'text');
+  docElement.className += ' toggle-pw';
```

Comment:
IE10 had its own password button to show / hide passwords, so this might still be useful.